packages/client/browser/libp2pnode.ts

```diff
@@ -3,9 +3,13 @@
  * @memberof module:net/peer
  */
 
+// @ts-ignore TODO: fix could not find a declaration file (browser)
```

Comment:
I think for these TODO's we also want to put this into a tracking issue, otherwise they tend to be forgotten until someone stumbles upon these TODO's in the files.

Reply:
Added to #975